addons/l10n_br/data/l10n_br_chart_data.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 	<record id="l10n_br_account_chart_template" model="account.chart.template">
-		<field name="name">Plano de Contas Brasileiro</field>
+		<field name="name">Brazilian Chart of Accounts</field>
```

Comment:
Could you please fix the indentation in this file (some lines use tabs and other spaces)?

Reply:
Oups sorry, I was not totally clear :sweat_smile: 
Please fix the indetation using spaces ahah
https://peps.python.org/pep-0008/#tabs-or-spaces
Btw, I know that's not Python here but to stay consistent we prefer using spaces instead of tabs!
Thank you